ui/core/components/suggest_reforges_action.tsx

```diff
@@ -540,6 +555,49 @@ export class ReforgeOptimizer {
 		);
 	}
 
+	setStatCaps(eventID: EventID, newStatCaps: Stats) {
+		this._statCaps = newStatCaps;
+		this.statCapsChangeEmitter.emit(eventID);
+	}
```

Comment:
We should do the same thing here as well: turn off soft cap breakpoints automatically if `newStatCaps` doesn't match the defaults from the sim config. This should reduce user errors where custom weights or custom caps are specified, but then the default breakpoints result in mismatched behavior for the Reforger.